src/web/landing/src/Flow/Website/Controller/DefaultController.php

```diff
@@ -2,120 +2,60 @@
 
 namespace Flow\Website\Controller;
 
+use function Flow\ETL\Adapter\JSON\to_json;
+use function Flow\ETL\DSL\df;
+use function Flow\ETL\DSL\lit;
+use function Flow\ETL\DSL\not;
+use function Flow\ETL\DSL\ref;
+use Flow\ETL\Adapter\Http\PsrHttpClientDynamicExtractor;
+use Flow\ETL\Filesystem\SaveMode;
+use Flow\Website\Factory\Github\ContributorsUrlFactory;
+use Http\Client\Curl\Client;
+use Nyholm\Psr7\Factory\Psr17Factory;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
 
-class DefaultController extends AbstractController
+final class DefaultController extends AbstractController
 {
+    private readonly string $path;
+
+    public function __construct(
+        private readonly ContributorsUrlFactory $contributorsUrlFactory
+    ) {
+        $this->path = __DIR__ . '/../../../../var/data/latest_contributors.json';
+    }
+
     #[Route('/', name: 'main')]
-    public function main() : Response
+    public function __invoke() : Response
```

Comment:
I think main was ok, we are also going to add few other pages later like for example something that will read documentation from `/docs` folder and dump it as static page or `installation wizard` that will help determine which packages user needs based on his answer